async-request-reply/src/AsyncProcessingBackgroundWorker.cs

```diff
@@ -10,19 +10,19 @@ public static class AsyncProcessingBackgroundWorker
     {
         [FunctionName("AsyncProcessingBackgroundWorker")]
         public static void Run(
-            [ServiceBusTrigger("outqueue", Connection = "ServiceBusConnectionAppSetting")]Message myQueueItem, 
+            [ServiceBusTrigger("outqueue", Connection = "ServiceBusConnectionAppSetting")]ServiceBusMessage myQueueItem,
             [Blob("data", FileAccess.ReadWrite, Connection = "StorageConnectionAppSetting")] CloudBlobContainer inputBlob,
```

Comment:
I don't think this type will work with v5 of the storage extension. Can you test it?

See https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs#writing-byte-array-to-blob